lib/traject/pdc_describe_indexing_config.rb

```diff
@@ -125,203 +125,19 @@
   accumulator.concat domains
 end
 
-# # ==================
-# # contributor fields
```

Comment:
I went ahead and deleted a ton of commented lines of fields that by now it's clear we are not going to need. If we need individual fields we can add them one by one as needed.